my-profile-middleware-impl/src/main/java/edu/wisc/my/profile/service/LocalContactInformationServiceImpl.java

```diff
@@ -58,8 +58,13 @@ public ContactInformation getContactInfo(String username, String manifestGroups,
 
   @Override
   public List<User> getUsers(String username, String manifestGroups, String searchTerm) {
-    // TODO implement this once method is provided
-    return null;
+      if(StringUtils.isNotBlank(adminGroup) && manifestGroups.contains(adminGroup)) {
+          logger.info("User {} looked up users using this search term: {}.",username, searchTerm);
+          return dao.getUsersBySearchTerm(searchTerm);
+      } else {
+          //TODO : implement uw-spring-security
+          throw new SecurityException("Security violation, forbidden.");
+      }
```

Comment:
Ahhh, now I see why you put it in LC dao. either way, works for me. It would be logical to make a "search" service ya?